.github/workflows/wordpress-plugin-deploy.yml

```diff
@@ -13,8 +13,7 @@ jobs:
       uses: actions/checkout@v2
     - name: Build
       run: |
-        npm install
-        npm run build
+        composer install
```

Comment:
It looks like we have a similar build command in the [wordpress-plugin-asset-update.yml](https://github.com/10up/safe-svg/blob/develop/.github/workflows/wordpress-plugin-asset-update.yml#L14) file. Should that be updated as well?

Reply:
Thanks! I missed that other build command. That's been updated now!